### PR TITLE
feat: filter functionality

### PR DESCRIPTION
**Pull request by nakulsharma15**
Feat: Added filter functionality which consists of:
1. Filter by priority
2. Filter by label
3. Filter by date

### CODE REVIEW

File: src/Pages/Home.jsx [hunk 1]
```diff
@@ -37,17 +35,16 @@ export default function Home() {
                     </div> : null
                     }
 
-                    {(notes.length !== 0) ?
+                    {(filteredNoteList.length !== 0) ?
```
Comment (krishanu2911) [inline src/Pages/Home.jsx:38]: You can use an operator other than ternary operator

**Review by krishanu2911 — APPROVED** [review]
Well written code
- Nicely handled filter functionality
- Well written filter function
Just added a small suggestion

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good .
try to write less line of code .

File: src/Pages/Home.jsx (hunk 1 above)
Comment (nakulsharma15) [reply src/Pages/Home.jsx:38]: I'll Look into it. Thank you.

**Pull request merged**